catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/email/EmailNotifier.java

```diff
@@ -175,6 +179,7 @@ private Properties createSessionProperies() {
         Properties properties = System.getProperties();
 
         properties.setProperty(SMTP_HOST_PROPERTY, mailHost);
+        properties.setProperty("mail.smtp.auth", "false");
```

Comment:
nit: since the other property strings are defined as constants, it would be consistent to have "mail.smtp.auth" defined as a constant.